src/tools/ceph_monstore_tool.cc

```diff
@@ -532,6 +532,13 @@ static int update_auth(MonitorDBStore& st, const string& keyring_path)
     KeyServerData::Incremental auth_inc;
     auth_inc.name = k.first;
     auth_inc.auth = k.second;
+    if (auth_inc.auth.caps.empty()) {
```

Comment:
i don't want to fill `auth.cap` with the following caps if it is empty, instead, it would be easier for user to customize the behavior of this tool by pointing it to the right keyring file, in which, keys like
```
[mon.]
    key = AQBZKO5XD71DFhAAKM6WEdA94RgqzHcX5+Eajw==
    caps mon = "allow *"
[client.admin]
    key = AQBZKO5XPZJyFxAAJ0B5//6kYxhmxSw1rcciWw==
    auid = 0
    caps mds = "allow *"
    caps mon = "allow *"
    caps osd = "allow *"
```
are expected.